portfolio/src/main/webapp/projects.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<head>
+  <meta charset="UTF-8">
+  <title>Projects</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="script.js"></script>
+</head>
+<body>
+  <div id="content">
+    <h1>My Projects</h1>
+    <p>Here is a list of projects that I've completed, ordered chronologically:
+
+    <h3>All-in</h3>
+    <h5>iOS app designed to build community around live events</h5>
+    <p>This is my most recent project, something dreamed up by my partner and I after a few too many weeks in iso. We both learnt Swift to work on this project! 
+
+    <h3>Open Data Crime Map</h3>
+    <h5>An interactive map of every crime committed on campus</h5>
+    <ul>
+      <li>Flask app that scrapes Harvard Police Department PDF crime logs using Pyton, scheduled using Cron.
+      <li>Store data using Firebase and Google App Engine.
+      <li>Display crimes as colour-coded 'hotspots' on an interactive map using Google Maps API.
+    </ul>
+
+    <h3>B-nook</h3>
+    <h5>An Ebook reader that makes reading in a foreign language simple</h5>
+    <ul>
+      <li>Built using React Native: integrated Microsoft Azure API for inline translation, set-up Firebas backend for Ebook storage, built vocab saving and self-testing features.
+      <li>Designed and maintained a landing page using Bootstrap. 
+      <li>Selected for Harvard's Startup R&D class
+    </ul>
+  </div>
+
+  <!-- Navigation bar -->
+  <ul class="nav-bar">
+      <li><a href="projects.html">My Projects</a>
+      <li><a href="gallery.html">Image Gallery</a>
+      <li><a href="https://github.com/AshleyLTW">Github</a>
+      <li><a href="https://www.linkedin.com/in/ashleywong99/">LinkedIn</a>
+  </ul>
+</body>
```

Comment:
nit: just make sure you have trailing newlines in your files, otherwise git shows an ugly icon here :(